.storybook/foundations/down-state.mdx

```diff
@@ -0,0 +1,24 @@
+import { Meta, Story } from '@storybook/blocks';
+import * as ButtonStories from '../../components/button/stories/button.stories.js';
+import * as CheckboxStories from '../../components/checkbox/stories/checkbox.stories.js';
+
+<Meta title="Foundations/Down state" />
+
+<h1>Down state</h1>
+
+Down state is a Spectrum 2 (S2) feature that creates the illusion of components being pressed-in when active. This functionality is already included in S2 components that require down state in this project. It is implemented with a CSS transform. The implementation depends on the size of the interactable element:
+
+  - If the interactable element has a width of 24px or less, we use the minimum perspective token:
```

Comment:
Formatting-wise for this section, I'd drop the bullet points, and add an "Examples" heading above the stories. 

Another idea would be to have two separate headings for the minimum perspective and the max formula, with each relevant story moved under each. 

It might also be worth noting the values of the two custom variables being set on the button stories in text above it.

Reply:
> It might also be worth noting the values of the two custom variables being set on the button stories in text above it, since those are only visible if you use the inspector.

I'm not clear on this, do you mean the width and height tokens?

Reply:
Yeah, I was thinking that it could say something like:

In this example, the width and height tokens are set to the following with an inline style: `--spectrum-downstate-width:72px; --spectrum-downstate-height: 32px;`